src/port1.0/portutil.tcl

```diff
@@ -3336,12 +3336,17 @@ proc _check_xcode_version {} {
             10.15 {
                 set min 11.0
                 set ok 11.3
-                set rec 11.4.1
+                set rec 11.5
+            }
+            10.16 {
+                set min 11.0
+                set ok 11.3
+                set rec 11.5
```

Comment:
Surely for 10.16/11 the min/ok/rec should be 12.0?

Reply:
My guess is,
(diff)
- 10.16
	- `set rec 12.0`
- 11.x
	- `set min 12.0`
	- `set ok 12.0`
	- `set rec 12.x`